### ISSUE TITLE
Product referential integrity check throws invalid WARNINGs

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/validate#368]
## 🐛 Describe the bug

When I try to validate against the [insight documents bundle](https://pds.nasa.gov/data/pds4/test-data/registry/urn-nasa-pds-insight_documents/) (as an example), I am getting WARNINGs about LID references not found for products that do not pertain to this bundle.

```
      WARNING  [info.validation.general]   A LID reference urn:nasa:pds:context:instrument:seis.insight is referencing a logical identifier for a product not found in this bundle
```

## 📜 To Reproduce

```
$ validate-2.1.0-SNAPSHOT/bin/validate -t /data/pds4/test-data/registry/urn-nasa-pds-insight_documents/bundle_insight_documents.xml -R pds4.bundle
...
  PASS: file:/data/pds4/test-data/registry/urn-nasa-pds-insight_documents/document_hp3rad/release_notes.xml
      WARNING  [info.validation.general]   A LID reference urn:nasa:pds:context:instrument:hp3.insight is referencing a logical identifier for a product not found in this bundle
...
```

## 🕵️ Expected behavior

- [ ] Validate should ignore these references for the time being. We can throw an INFO or DEBUG message here if we want, and then eventually replace with some registry integration, but not now. This will definitely dramatically inflate the number of warning messages coming across (especially for context products).
- [ ] For WARNINGs that should exist, the problem type is incorrect. Current problem type: `info.validation.general`. Replace with new problem type: `warning.integrity.reference_not_found`

## 📚 Version of Software Used

2.1.0-SNAPSHOT

## 🩺 Test Data / Additional context

Really any bundle, but https://pds.nasa.gov/data/pds4/test-data/registry/urn-nasa-pds-insight_documents/ also

## 🖥 System Info

---

## :unicorn: Applicable requirements
🦄 #308

**Comment by tloubrieu-jpl** [comment]
@qchaupds will branch off #69 for this ticket ()

**Comment by jordanpadams** [comment]
@qchaupds per this ticket, just added another update that should be include here:

> For WARNINGs that should exist, the problem type is incorrect. Current problem type: `info.validation.general`. Replace with new problem type: `warning.integrity.reference_not_found`

**Comment by qchaupds** [comment]
Because the code from branch #69 has been merged back into main branch on July 17, 2021, the work for this ticket will be branched off a fresh copy of main.

**Comment by jordanpadams** [comment]
closed per #383

**Comment by rchenatjpl** [comment]
@jordanpadams @qchaupds
Am I missing something? This is the same as #308, right?

**Comment by jordanpadams** [comment]
@rchenatjpl this was a bug identified after #308 was implemented. as long as you feel #308 was testing sufficiently, I think we can say this was tested as well
